SimPEG/electromagnetics/viscous_remanent_magnetization/simulation.py

```diff
@@ -917,8 +917,8 @@ def fields(self, m):
         self.model = m  # Initiates/updates model and initiates mapping
 
         # Project to active mesh cells
-        # m = np.matrix(self.xiMap * m).T
-        m = np.matrix(self.xiMap * m).T
+        # m = np.atleast_2d(self.xiMap * m).T
+        m = np.atleast_2d(self.xiMap * m).T
```

Comment:
I believe you should just be able to do...
` m = self.xiMap * m` (no need to convert it to anything at all)

Reply:
Done, the tests should show if it works :-)